### PR TITLE
Add contributing section in README

### PR DESCRIPTION
**Pull request by kaustubhhiware**
In continuation to the discussion at #20 . Edits and suggestions welcomed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,3 +82,7 @@ Want to uninstall and revert back to the old style?
 ```sh
 gem uninstall colorls
 ```
+
+# Contributing
+
+Your contributions are always welcome! Please take a look at the [contribution guidelines](CONTRIBUTING.md) first.
```
Comment (athityakumar) [inline README.md]: take -> have. Add a 🎉  at the end maybe? 😄

Comment (kaustubhhiware) [inline README.md]: You really are obsessed with :tada: , aren't you? Will update

Comment (kaustubhhiware) [inline README.md]: At the end of the first sentence or the second ?

Comment (athityakumar) [inline README.md]: At the end of second sentence. 🎉

Comment (kaustubhhiware) [inline README.md]: Done :tada:

**Pull request merged**